lv_conf_template.h

```diff
@@ -709,8 +709,13 @@ typedef void * lv_obj_user_data_t;
 #define LV_USE_TABLE    1
 #if LV_USE_TABLE
 #  define LV_TABLE_COL_MAX    12
+#  define LV_TABLE_CELL_STYLE_CNT 4
+#  if (LV_TABLE_CELL_STYLE_CNT > 16)
+#    error LV_TABLE_CELL_STYLE_CNT cannot exceed 16
```

Comment:
Typically we put conditions like these in the widget's header file to keep the logic out of the configuration template.

Reply:
It is in the header file as well as the conf_template. I thought it may be useful to notify the user at source of the error? If you wish I can make another submit with the check removed in conf_template, but retained in header_file

Reply:
Yes; I think that would be better. `lv_conf_template.h` gets processed by scripts as well so it's somewhat important to keep it consistent.

You could move the `#error` to the header file but leave a comment here about the fact that the value must be <= 16.

Reply:
Made a new commit with the check removed from lv_conf_template.h

Reply:
The check is there in lv_table.h